tests/test_klaviyo_automatic_fields.py

```diff
@@ -18,8 +18,10 @@ def test_run(self):
         • Verify that only the automatic fields are sent to the target.
         • Verify that all replicated records have unique primary key values.
         """
-        # We are not able to generate test data so skipping two streams(mark_as_spam, dropped_email)
-        expected_streams = self.expected_streams() - {"mark_as_spam", "dropped_email"}
+
+        # Test account does not have data for untestable streams
+        untestable_streams = {"unsubscribe", "mark_as_spam", "dropped_email"}
```

Comment:
can we generate data for the above streams?

Reply:
We can't generate data for these streams, hence, they were put in the list of untestable streams.